packages/form-js-playground/test/TestHelper.js

```diff
@@ -1,6 +1,7 @@
 import './test.css';
 
 import '@bpmn-io/form-js-viewer/dist/assets/form-js.css';
+import '@bpmn-io/form-js-viewer/dist/assets/light.css';
```

Comment:
Just a reminder to rename this once we merge https://github.com/bpmn-io/form-js/pull/453